app/src/main/java/com/droidhats/campuscompass/viewmodels/CalendarViewModel.kt

```diff
@@ -15,19 +16,69 @@ class CalendarViewModel(application: Application) : AndroidViewModel(application
 
     private val context = getApplication<Application>().applicationContext
     private var userCalendars = MutableLiveData<MutableMap<String, Calendar>>()
+    private var selectedCalendars = MutableLiveData<ArrayList<Calendar>>()
 
-    private val _text = MutableLiveData<String>().apply {
-        value = "Calendar"
+    private val _info = MutableLiveData<String>().apply {
+        value = ""
     }
-    val text: LiveData<String> = _text
+    var info: MutableLiveData<String> = _info
```

Comment:
Why are you exposing a mutable variable here as opposed to an immutable one? This may cause bugs if not used carefully.

Reply:
Youre right I did this lazily to not have to create different strings for each status message. Im just changing this one.